utils/build/test.mk

```diff
@@ -45,15 +45,17 @@ STATIC_ENABLE=
 
 .PHONY: test
 test: $(T_TARGET)
-	$(Verb) ./$< --gtest_filter=-*_noci
+	$(Verb) ./$< --gtest_filter=-*_dev
 
+test-dev: $(T_TARGET)
+	$(Verb) ./$< --gtest_filter=*_dev
 test-all: $(T_TARGET)
```

Comment:
You should add .PHONY for test-dev, test-all and test-filter. This tells Make that these are not files to be build but rather *commands*.

Reply:
Will do. The naming isn't final on the "dev" so was going to gather opinions on what best to call it from QA and other teams. As for the differentiation tests these were more just subjects to ensure it worked correctly. I will remove their tags :)